eyelid-match-ops/src/primitives/poly/modular_poly/conf.rs

```diff
@@ -0,0 +1,57 @@
+//! Fixed parameters for modular polynomial types.
+
+use std::fmt::Debug;
+
+/// The polynomial config used in tests.
+//
+// We use the full resolution by default, but TinyTest when cfg(tiny_poly) is set.
+#[cfg(not(tiny_poly))]
+pub type TestRes = FullRes;
+
+/// The polynomial config used in tests.
+#[cfg(tiny_poly)]
+pub type TestRes = TinyTest;
+
+/// Fixed polynomial parameters.
+///
+/// Polynomials with different parameters are incompatible.
+pub trait PolyConf: Copy + Clone + Debug + Eq + PartialEq {
+    /// The maximum exponent in the polynomial.
+    const MAX_POLY_DEGREE: usize;
+
+    // TODO: add Coeff type
```

Comment:
Yes, coefficient modulus will be important to be defined here for each different configuration.

Reply:
That's next, see the checklist in ticket #59